src/IO/H5/Dat.cpp

```diff
@@ -107,6 +109,7 @@ void Dat::append_impl(const hsize_t number_of_rows,
     }
     CHECK_H5(H5Sclose(dataspace_id), "Failed to close dataspace");
     if (read_size != size_) {
+      using ::operator<<;
```

Comment:
Because ADL doesn't find stream operators for containers if they are not in their namespace and putting stream operators (or almost anything for that matter) into the `std` namespace is undefined behavior. This pulls in all global stream operators into the current scope so that ADL can find them.